extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -166,6 +171,20 @@ export class AmpSlideScroll extends BaseSlides {
     return this.noOfSlides_ > 2;
   }
 
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    mutations.forEach(mutation => {
+      const newValue = mutation.newValue;
+      switch (mutation.name) {
+        case 'slide':
+          const slide = parseInt(newValue, 10);
+          user().assert(isFinite(slide), 'Invalid [slide] value: %s', newValue);
+          this.showSlide_(slide);
```

Comment:
because this was invoked by another event and we would not want to have an infinite event cycle?